src/main/proto/firrtl.proto

```diff
@@ -34,9 +34,15 @@ message Firrtl {
     oneof source_info {
```

Comment:
@sdtwigg if you have a chance, I would appreciate your looking at this, especially as it relates to possible integration with your representation (although I believe you mentioned y'all are using proto2 so maybe that just won't work?)